README.md

```diff
@@ -49,3 +49,19 @@ A rinkeby example for the DAO was deployed, with its Avatar on address [0x5B3dd2
 3. Update `prepare` cmd at [package.json](https://github.com/daostack/nectardao/blob/master/package.json#L9)
 4. `npm i`
 5. `npm run migrate --  --private-key <PRIVATE_KEY> --provider https://mainnet.infura.io/v3/<infuraKey>`
+
+## Redemption Script
+
+In order to run the redeeming script, start by adding the following arguments to your `.env` file.
+
+```bash
+private_key=YOUR_PRIVATE_KEY
+ws_provider=YOUR_WEB_SOCKER_WEB3_PROVIDER
+from_block=THE_BLCOK_TO_START_SCANNING_FROM
+gas_price=GAS_PRICE_FOR_TXS
+NectarReputationRedeemerAddress=REDEEMING_CONTRACT_ADDRESS
```

Comment:
this is the batcher contract ?
we can keep its address hard coded .no?

Reply:
Well we can but then it will work only on mainnet.

Reply:
we can have it also on Rinkeby